examples/light-switch-app/nrfconnect/main/AppTask.cpp

```diff
@@ -487,6 +490,14 @@ void AppTask::ButtonEventHandler(uint32_t aButtonState, uint32_t aHasChanged)
         sAppTask.PostEvent(&buttonEvent);
     }
 
+    if (GROUP_SWITCH_BUTTON_MASK & aHasChanged & aButtonState)
+    {
+        buttonEvent.ButtonEvent.PinNo  = GROUP_SWITCH_BUTTON;
+        buttonEvent.ButtonEvent.Action = AppEvent::kButtonPushEvent;
+        buttonEvent.Handler            = ButtonReleaseHandler;
```

Comment:
Shouldn't we use `ButtonPushHandler`? If the `aButtonState` is true that mean the button is pressed? Also why do we have push and release handlers for unicast and only one for groupcast? Are we going to not support dimming for groupcast?